Dockerfile

```diff
@@ -13,6 +13,14 @@ WORKDIR /opt/evm_loader/program
 RUN cargo build-bpf --features no-logs
 RUN cd ../cli && cargo build --release
 
+# Download and build spl-token
+FROM builder AS spl-token-builder
+ADD http://github.com/solana-labs/solana-program-library/archive/refs/tags/token-cli-v2.0.11.tar.gz /opt/
+RUN tar -xvf /opt/token-cli-v2.0.11.tar.gz
+WORKDIR /opt/solana-program-library-token-cli-v2.0.11/token/cli
+RUN cargo build --release
+RUN mv /opt/solana-program-library-token-cli-v2.0.11/target/release/spl-token /opt/
```

Comment:
It's better to merge this `RUN` into one. WORKDIR can be removed (use `cd ..` instead).
```
RUN tar -xvf /opt/token-cli-v2.0.11.tar.gz && \
    cd /opt/solana-program-library-token-cli-v2.0.11/token/cli && \
    cargo build --release && \
    cp target/release/spl-token /opt
```

Reply:
Done